02.calendar/calender.rb

```diff
@@ -0,0 +1,33 @@
+require 'date'
+require 'optparse'
+
+year = Date.today.year
+mon = Date.today.mon
+params = ARGV.getopts("m:y:")
+
+if params["m"].nil?
+else
+  mon = params["m"].to_i
+end
+if params["y"].nil?
+else
+  year = params["y"].to_i
+end
+
+title = "#{mon}月 #{year}"
+firstday_wday = Date.new(year,mon,1).wday
+lastday_date = Date.new(year,mon,-1).day
+
+puts title.center(20)
+puts "日 月 火 水 木 金 土"
+print "   " * firstday_wday
+(1..lastday_date).each do |date|
+  print date.to_s.rjust(2) + " "
+  firstday_wday = firstday_wday+1
```

Comment:
これもrubocopを使い始めれば機械的に指摘されると思いますが、演算子の前後は半角スペースを空けた方が良いです。27行目、31行目なども同様です。
```suggestion
  firstday_wday = firstday_wday + 1
```